site/content/docs/5.2/components/close-button.md

```diff
@@ -45,6 +45,38 @@ Close buttons can also be created without `.btn-close`.
   <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false"><use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#delete"></use></svg>
   <span class="visually-hidden">Close</span>
 </button>
+
+<button type="button" class="btn btn-icon btn-secondary">
```

Comment:
Where do we use those buttons in the DSM?

Reply:
In the [variant section of the DSM](https://system.design.orange.com/0c1af118d/p/278ebc-buttons-standard/b/247486/t/12ce14). Do you think we should add a variant like `.btn-close-bordered` for the `.btn-close`?

Reply:
You should ask this to the designers. I'm not sure to have seen components using this variant as a close button. IMO the link you're referring to is only an example of an outline button with an icon in it; it doesn't necessarily mean we should insert it as a "Close button".

Reply:
Ah you might be right, it's a bit confusing, I'll ask as soon as possible to be clarified on this point!

Reply:
Will be merged with #1705.